src/components/Votes.jsx

```diff
@@ -1,44 +1 @@
-import { patchReviewVotes } from "../requests";
-import { useState } from "react";
 
-export default function Votes({ review_id, review }) {
-  const [voteCount, setVoteCount] = useState(review.votes);
-  const [upDisabled, setUpDisabled] = useState(false);
-  const [downDisabled, setDownDisabled] = useState(false);
-  const [err, setErr] = useState(null);
-
-  const handleClickUp = () => {
-    setVoteCount((currCount) => currCount + 1);
-    setErr(null);
-    patchReviewVotes(review_id, 1).catch(() => {
-      setVoteCount((currCount) => currCount - 1);
-      setErr("something went wrong please try again");
-    });
-    setUpDisabled(true);
-    setDownDisabled(false);
-  };
```

Comment:
You're handling what to do with patchReviewVotes returns an error well here in terms of the Count, but do we not also want to change what upDisabled and downDisabled are for errors?